pkgs/development/python-modules/tellduslive/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, docopt
+, fetchFromGitHub
+, requests
+, requests_oauthlib
+, isPy39
+}:
+
+buildPythonPackage rec {
+  pname = "tellduslive";
+  version = "0.10.11";
+
+  src = fetchFromGitHub {
+    owner = "molobrakos";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0aqhj6fq2z2qb4jyk23ygjicf5nlj8lkya7blkyqb7jra5k1gyg0";
+  };
+
+  propagatedBuildInputs = [
+    docopt
+    requests
+    requests_oauthlib
+  ];
+
+  # Project has no tests
+  doCheck = false;
+  # Install check fails on Python 3.9, RuntimeWarning: 'testtools.run' found in sys.modules
+  doInstallCheck = !isPy39;
```

Comment:
Does this otherwise work on 3.9?

Reply:
Will work on 3.9 as soon as #112854 lands.